packages/cactus-test-tooling/src/main/typescript/fabric/fabric-test-ledger-v1.ts

```diff
@@ -301,6 +313,7 @@ export class FabricTestLedgerV1 implements ITestLedger {
 
   public async start(omitPull: boolean = false): Promise<Container> {
     const containerNameAndTag = this.getContainerImageName();
+    const containerFabricVersion = this.getFabricVersion();
```

Comment:
Here you can do the `Map<string,string>` to `string[]` conversion instead.

Reply:
Incorporated the changes. Please have a re-look.